R/registry.R

```diff
@@ -34,14 +34,17 @@ is_credential_function <- function(f) {
 
 #' Add a new credential fetching function.
 #'
-#' Note that this implicitly adds `f` to the *end* of the list.
+#' Note that this implicitly adds `f` to the *front* of the list.
 #'
 #' @param f A function with the right signature. See [is_credential_function()].
 #' @family registration
 #' @export
 add_credential_function <- function(f) {
   stopifnot(is_credential_function(f))
   gargle_env$credential_functions <- c(f, gargle_env$credential_functions)
+  if (is.symbol(substitute(f))) {
```

Comment:
Reveal the names of the credential functions.

Hmm, I thought I posted an example. I guess not. About to do so.